en/focus_privacy_notice.md

```diff
@@ -1,24 +1,29 @@
 
+Read the telemetry documentation for [Android](https://dictionary.telemetry.mozilla.org/apps/focus_android) and [iOS](https://www.google.com/url?q=https://dictionary.telemetry.mozilla.org/apps/focus_ios/&sa=D&source=editors&ust=1633545718722000&usg=AOvVaw0Qsl1DMom9lNWMmPN1taVF).
 
-* __Interaction Data__: We receive information about how often you use Focus and interact with certain features or prompts. This includes information such as when you start a new browsing session, how often you use search within Focus or launch an external browser from Focus, and which features in the Settings screen are enabled.
+### Measure and support our marketing
 
+* __Campaign Data and Referral Data__: This helps Firefox Focus understand the effectiveness of our marketing campaigns. Focus may use “Referral Data” such as the website domain or advertising campaign that referred you to download and install Focus. This data is kept by [Adjust](https://www.adjust.com/terms/privacy-policy/), our analytics vendor, and provided when you start Focus, and also includes an iOS Vendor ID or Google advertising ID, timestamp, country, locale, operating system, and app version. Mozilla does not receive this data for Klar. Read the [documentation](https://github.com/mozilla-mobile/focus-android/wiki/Adjust-Usage). 
 
-* __Referral and Campaign Data__: To help improve our marketing campaigns, Focus may use “Referral Data” such as the website domain or advertising campaign that referred you to download and install Focus. This data is kept by our analytics vendor and provided when you start Focus, and also includes an iOS Vendor ID or Google advertising ID, timestamp, country, locale, operating system, and app version.
+### Support Search
 
-Learn more about [Technical and Interaction Data](https://github.com/mozilla-mobile/focus/wiki/Event-Tracking-with-Mozilla's-Telemetry-Service), [Referral and Campaign Data](https://github.com/mozilla-mobile/focus/wiki/Install-Tracking-in-Firefox-Focus-with-the-Adjust-SDK), or how to [opt-out](https://support.mozilla.org/kb/send-anonymous-usage-data-firefox-mobile-devices) of this data collection.
+You can perform searches directly in the Awesome Bar. Mozilla does not receive your search queries. We do receive data about how you engage with search in Firefox Focus and the number of searches you request from our search partners. Query data is sent to your default search provider, which has its own privacy policy. 
 
+* __Search suggestions__: If enabled, Firefox Focus sends search queries to your search provider to help you discover common phrases other people have searched for and improve your search experience. These data will not be sent if your selected search provider does not support search suggestions.
```

Comment:
I'm not sure about the _These data will not be sent if your selected search provider does not support search suggestions_ part - if the provider does not support suggestions, we still do send it the search term of course. It is a bit ambiguous. We don't call the suggestions API but we do send your search queries to the same search provider.

Reply:
Who can confirm if this is true or not?